aicsimageio/tests/readers/test_czi_reader.py

```diff
@@ -20,39 +20,39 @@
     "chunk_dims",
     [
         (
-            "s_1_t_1_c_1_z_1.czi",
+            "s_1_t_1_c_1_z_1.czi",  # noqa: E126
```

Comment:
I think these can be removed?

Reply:
The `noqa` flags that is

Reply:
Sure but I couldn't figure out how to get rid of the linting error associated with E126. 
How would you format these blocks?

Reply:
I am going to assume that something went wrong with this during the rebase because those lines are already on master and the lint checks pass.